### ISSUE TITLE
Login: inline error not announced on iOS safari browser

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#37]
URL: https://hcdev.openplus.ca/en/user/login
User Agent: Safari iOS / VoiceOver
Language: English

Issue Description: inline error not being announced by screen reader 
Steps to reproduce:
in safari tab, open link
turn Voice over on.
from the login page, leaving both fields blank, navigate to the Submit button
Activate the Submit button.
Error is displayed "fill out this field" but it not announced to screen reader. 
the focus is also set to the formatting options on top toolbar
see video 
[error not being read.zip](https://github.com/mmiron-HC/WST-RSAMS/files/6938637/error.not.being.read.zip)

Expected results:
Error message should be announced to user.

**Comment by echouin** [comment]
OpenPlus comments: Closing this. This is the Drupal default. Considering content management on Safari ios is likely not going to happen.

**Comment by echouin** [comment]
@mmiron-HC from OP: Safari not supported for login. Suggest this item be closed.

**Comment by mmiron-HC** [comment]
@echouin We will keep the issue open as this is still a failure.

**Comment by echouin** [comment]
@mmiron-HC From OP: "Safari IOS is not supported for login.":

**Comment by echouin** [comment]
@mmiron-HC Since this browser is not supoorted, can we close this out?

**Comment by mmiron-HC** [comment]
@echouin how are you going to confirm that they will not open this in safari? do you have a disclaimer that you cannot use this browser?
The [Guidance on implementing Web Standards](https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/guidance-implementing-standard-web-accessibility.html) says "The current and previous versions of the default browsers of each mobile operating systems that accounts for at least 5 per cent of the global or Canadian mobile operating system market share." under #8 Test Web pages with different browsers.

**Comment by echouin** [comment]
@mmiron-HC Hi Manon, this is specifically for CMS Users. I don't think that many of our CMS users would be adding or updating recall information to the RSA system through their Mobile devices. I don't believe this data-entry mode was requested by our CMS users either. I will also make a note in our CMS User HELP GUIDE that speaks to this fact. This is a feature we could look at implementing down the road but at this time, mobile CMS use would be extremely low.

**Comment by echouin** [comment]
This is now marked as Resolved for the following reasons:
1. Safari iOS is not supported for login.
2. No RSA CMS users should be using a smartphone device to login to the RSA CMS to add/update alerts/recalls to the system.
2. The RSA CMS login page is for RSA CMS users only and is not intended to be used by the general public.

**Comment by mmiron-HC** [comment]
@echouin This is still a failure. We've moved this to Won't fix.

**Comment by echouin** [comment]
@mmiron-HC I think it's outside of scope for OP to fix. May I suggest "Reported to Drupal module owner"?

**Comment by mmiron-HC** [comment]
if it is a Drupal Limitation than add the label. Milestone can stay as a Won't fix
